### ISSUE TITLE
Check against annotated NIST database and facebook/casual-conversations

### ISSUE THREAD

**Issue by tadam98** [BSI-OFIQ/OFIQ-Project#6]
Hi,

NIST publised an annotated database with ICAO compliance challenges:
https://www.nist.gov/news-events/news/2019/12/nist-releases-data-help-measure-accuracy-biometric-identification
==> https://www.nist.gov/itl/iad/image-group/resources/biometric-special-databases-and-software
==> https://www.nist.gov/itl/iad/image-group/special-database-32-multiple-encounter-dataset-meds

After request in: https://nigos.nist.gov/datasets/sd32/request
This can be downloaded: sd32-MEDS_I-II.tar.gz

I believe that running OFIQ against these faces could be very interesting.
(which is what I am aiming to do after successful compilation).

Another extremely interesting database was published by facebook. It contains 45,000 one minute movies of various ethnicities, one-two-three faces in a movie and the persons are changing expressions, moving a bit, changing facial angles etc.
https://ai.meta.com/datasets/casual-conversations-downloads/ 

I believe it could be a very interting excercise to get the score and the reasons in case a face is not as perfect as it should be.

Best,
Mickey

**Comment by bbtams** [comment]
Hi Mickey,
I don't see what we need to fix, to close the issue.
Best,
Benjamin

**Comment by tadam98** [comment]
Hi Benjamin,

The expectation from the project, as discussed in Tallinn is to address the following challenge:

1. Today, when we enroll a face, we can any number of measured parameters between 15 and 32. And "someone's" software or "someone's" officer has to decide whether the fface is good enough for Enrollement or should the picture we taken again.
2. In NFIQ2 the algorithm gave predicion whether the fingerprint is "likely" to provide good matching results on a scale of 0-100.
3. With FaceQ we expect a Quality Score 0-100 and some threshold above which the face is expected to be good enough. There could be two thresholds, one for 1:1 and a higher one requited for 1:N. In case the face fails to cross the thershold, some "reason(s)" would be provided explaining what is the problem. Which could be results of the 15-32 measured parameters and an indication which one is not as expected. For example "there is an expression", "not looking forward..." etc.

Having (3) in mind, the NIST faces database is annotated and has, for example the eye distances, expressions, dirction of face and many more. I think it could be interesting to see whether FaceQ indication of measured parameters is close to NIST's.

Also, opertaing the software of the Facebook movies (if deeped permitted and you see the merit) allows us to see the same as the persons in the movies change expression, move the head etc.

We have a goal to be able to operate in video rate, checking the movies can also verify this goal.

In summary, the ticket can be closed by (a) response that it is taken into account or (b) necessity is not seen at this time.

Best,
Mickey
